src/sharedwithyou.cs

```diff
@@ -0,0 +1,399 @@
+		NSString MetadataTypeIdentifier { get; }
+
+		[Export ("identifier", ArgumentSemantic.Copy)]
+		NSObject/*<NSSecureCoding, NSCopying>*/ Identifier { get; }
+
+		[Export ("URL", ArgumentSemantic.Copy)]
+		NSUrl Url { get; }
+	}
+
+	interface ISWHighlightCenterDelegate {}
+
+	[NoWatch, TV (16,0), NoMac, iOS (16,0), MacCatalyst (16,0)]
+#if NET
+	[Protocol, Model]
+#else
+	[Protocol, Model (AutoGeneratedName = true)]
+#endif
+	[BaseType (typeof (NSObject))]
+	interface SWHighlightCenterDelegate
+	{
+		[Abstract]
+		[Export ("highlightCenterHighlightsDidChange:")]
+		void HighlightCenterHighlightsDidChange (SWHighlightCenter highlightCenter);
+	}
+
+	[NoWatch, TV (16,0), NoMac, iOS (16,0), MacCatalyst (16,0)]
+	[BaseType (typeof (NSObject))]
+	interface SWHighlightCenter
+	{
+		[Wrap ("WeakDelegate")]
+		[NullAllowed]
+		ISWHighlightCenterDelegate Delegate { get; set; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		NSObject WeakDelegate { get; set; }
+
+		[Export ("highlights", ArgumentSemantic.Copy)]
+		SWHighlight[] Highlights { get; }
+
+		[Static]
+		[Export ("highlightCollectionTitle")]
+		string HighlightCollectionTitle { get; }
+
+		[Static]
+		[Export ("systemCollaborationSupportAvailable")]
+		bool SystemCollaborationSupportAvailable { [Bind ("isSystemCollaborationSupportAvailable")] get; }
+
+		[Async]
+		[Export ("getHighlightForURL:completionHandler:")]
+		void GetHighlight (NSUrl urL, Action<SWHighlight, NSError> completionHandler);
+
+		[Export ("collaborationHighlightForIdentifier:error:")]
+		[return: NullAllowed]
+		SWCollaborationHighlight GetCollaborationHighlight (string collaborationIdentifier, [NullAllowed] out NSError error);
+
+		[Async]
+		[Export ("getCollaborationHighlightForURL:completionHandler:")]
+		void GetCollaborationHighlight (NSUrl url, Action<SWCollaborationHighlight, NSError> completionHandler);
+
+		[Export ("postNoticeForHighlightEvent:")]
+		void PostNoticeForHighlightEvent (ISWHighlightEvent @event);
+
+		[Async]
+		[Export ("getSignedIdentityProofForCollaborationHighlight:usingData:completionHandler:")]
+		void GetSignedIdentityProof (SWCollaborationHighlight collaborationHighlight, NSData data, Action<SWSignedPersonIdentityProof, NSError> completionHandler);
+	}
+
+
+	[NoWatch, NoMac, NoTV, iOS (16,0), MacCatalyst (16,0)]
+	[BaseType (typeof (UIViewController))]
+	[DisableDefaultCtor]
+	interface SWRemoveParticipantAlertController
+	{
+		[Static]
+		[Export ("alertControllerWithParticipant:highlight:")]
+		SWRemoveParticipantAlertController Create (SWPerson participant, SWCollaborationHighlight highlight);
+	}
+
+	interface ISWCollaborationViewDelegate {} 
+
+	[NoWatch, NoTV, NoMac, iOS (16,0), MacCatalyst (16,0)]
+#if NET
+	[Protocol, Model]
+#else
+	[Protocol, Model (AutoGeneratedName = true)]
+#endif
+	[BaseType (typeof (NSObject))]
+	interface SWCollaborationViewDelegate
+	{
+
+		[DesignatedInitializer]
+		[Export ("initWithFrame:")]
+		NativeHandle Constructor (CGRect frame);
```

Comment:
This needs to be inlined in the implementor classes 
```suggestion
		// [DesignatedInitializer]
		// [Export ("initWithFrame:")]
		// NativeHandle Constructor (CGRect frame);
```